sheetstorm/management/commands/data_load.py

```diff
@@ -0,0 +1,278 @@
+
+User = get_user_model()
+
+
+class Command(BaseCommand):
+    help = "Create initial sample data for SheetStorm application testing."
+
+    def add_arguments(self, parser):
+        #Users
+        parser.add_argument('-a','--admins', type=int, help='Indicates the number of admins to be created')
+        parser.add_argument('-e','--employees', type=int, help='Indicates the number of employees to be created')
+        parser.add_argument('-m','--managers', type=int, help='Indicates the number of managers to be created')
+        #Projects
+
+    @transaction.atomic
+    def handle(self, *args, **options):
+        #
+        # Users
+        #
+        self.stdout.write(f"Number of users in db before add {User.objects.count() }")
+        super_admin = User.objects.filter(email=f"admin@codepoets.it").first()
+        if super_admin is None:
+            super_admin = UserFactory(
+                email=f"admin@codepoets.it", user_type=User.UserType.ADMIN.name, is_superuser=True, is_staff=True,
+                first_name="Admin",
+                last_name="Kowalski"
+            )
+            super_admin.set_password("superuser")  # pylint: disable=no-value-for-parameter
+            super_admin.save()
+            self.stdout.write(f"Add super user")
+
+        number_of_admins_to_create = options['admins']
+        if number_of_admins_to_create:
+            for new_admin in range(number_of_admins_to_create):
+                user_admin = User.objects.filter(email=f"user1_{new_admin}@codepoets.it").first()
+                if user_admin is None:
+                    user_admin = UserFactory(
+                        email=f"user1_{new_admin}@codepoets.it", user_type=User.UserType.ADMIN.name, is_superuser=False, is_staff=True,
+                        first_name=f"User1_{new_admin}",
+                        last_name="Normal-Admin"
+                    )
+                    user_admin.set_password("admin1234")  # pylint: disable=no-value-for-parameter
+                    user_admin.save()
+                    self.stdout.write(f"Add normal admin {new_admin+1}")
+        else:
+            pass
+
+        number_of_employees_to_create = options['employees']
+        if number_of_employees_to_create:
+            for new_employee in range(number_of_employees_to_create):
+                user_employee = User.objects.filter(email=f"User2_{new_employee}@codepoets.it").first()
+                if user_employee is None:
+                    user_employee = UserFactory(
+                        email=f"User2_{new_employee}@codepoets.it",
+                        user_type=User.UserType.EMPLOYEE.name,
+                        first_name=f"User2_{new_employee}",
+                        last_name="Employee",
+                    )
+                    user_employee.set_password("passwduser")  # pylint: disable=no-value-for-parameter
+                    user_employee.save()
+                    self.stdout.write(f"Add employee {new_employee+1}")
```

Comment:
The logs are a great idea 👍 , but you should use an actual logger instead of **stdout.write**. We've been using a logger available from **logging** module for this exact purpose. Please use it everywhere where you're using **stdout.write**

Reply:
done